openshift/template.yaml

```diff
@@ -210,6 +213,8 @@ objects:
                 value: ${ALLOWED_DOMAINS}
               - name: OCM_BASE_URL
                 value: ${OCM_BASE_URL}
+              - name: OCM_DEBUG
```

Comment:
Can you change the value to be log level instead of boolean?

Reply:
the default can be info or whatever it's now

Reply:
sure, changed.